packages/askar/src/wallet/AskarWallet.ts

```diff
@@ -364,7 +364,7 @@ export class AskarWallet implements Wallet {
         const key = seed ? AskarKey.fromSeed({ seed: Buffer.from(seed), algorithm }) : AskarKey.generate(algorithm)
 
         // Store key
-        await this.session.insertKey({ key, name: encodeToBase58(key.publicBytes) })
+        await this.session.insertKey({ key, name: utils.encodeToBase58(key.publicBytes) })
```

Comment:
Can't we use the TypedArrayEncoder for this?

Reply:
Yeah I think so, I will use it directly.